### ISSUE TITLE
Can’t install @stryker-mutator/mocha-runner with npm v7

### ISSUE THREAD

**Issue by lo1tuma** [stryker-mutator/stryker#2606]
I get the following error when I try to install the latest version `4.1.0` of `@stryker-mutator/mocha-runner` with npm v7:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: my-project@0.0.0-dev
npm ERR! Found: @stryker-mutator/core@4.1.0
npm ERR! node_modules/@stryker-mutator/core
npm ERR!   dev @stryker-mutator/core@"4.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @stryker-mutator/core@"~4.0.0" from @stryker-mutator/mocha-runner@4.1.0
npm ERR! node_modules/@stryker-mutator/mocha-runner
npm ERR!   dev @stryker-mutator/mocha-runner@"4.1.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

It seems like `@stryker-mutator/mocha-runner` is specifying a peer dependency to `@stryker-mutator/core` version < `4.1.0`.

**Comment by nicojs** [comment]
Hi @lo1tuma, thanks for stopping by 🙋‍♂️

4.1 is our latest release. We lock our versions, so version 4.1 of the `@stryker-mutator/mocha-runner` should be installed together with version 4.1 of `@stryker-mutator/core`. Could you please install that version and try again? https://www.npmjs.com/package/@stryker-mutator/core

**Comment by lo1tuma** [comment]
Hi @nicojs 

thanks for the response. But I’m already using version `4.1` of `@stryker-mutator/core`.
Here is the section of my `package.json`

```
        "@stryker-mutator/core": "4.1.0",
        "@stryker-mutator/mocha-framework": "4.1.0",
        "@stryker-mutator/mocha-runner": "4.1.0",
        "@stryker-mutator/typescript": "4.1.0",
```

The problem is that `@stryker-mutator/mocha-framework` in version `4.1` is not compatible to `@stryker-mutator/core` in `4.1` since it declares a peerDependency to `@stryker-mutator/core` in version `~4.0.0`, see [here](https://github.com/stryker-mutator/stryker/blob/6bd2fbbf0aaa5154930ce36f4d153ee91a3c5f1f/packages/mocha-runner/package.json#L51).
I think the `package.json` of `@stryker-mutator/mocha-framework`  needs to be changed to require a peerDependency of `@stryker-mutator/core` `4.1`.

**Comment by nicojs** [comment]
Ah wow. Didn't notice this. Thanks for the issue, will fix

**Comment by nicojs** [comment]
Fixed in 4.1.2

**Comment by lo1tuma** [comment]
Thank you 🍻
